src/webui.c

```diff
@@ -77,11 +77,37 @@ fetch_wa_cookie(struct evhttp_request *req)
     return wa;
 }
 
+static void
+send_headers(struct evhttp_request *req, const char *contype)
+{
+    struct evkeyvalq *hdrs_out = NULL;
+    hdrs_out = evhttp_request_get_output_headers(req);
+    evhttp_add_header(hdrs_out, "Content-Type", contype);
+}
+
+static void
+send_json_worker(struct evhttp_request *req, void *arg, const char *rig_id)
+{
+    struct evbuffer *buf = evhttp_request_get_output_buffer(req);
+    double mh[6] = {0};
+    const char *wa = fetch_wa_cookie(req);
+    if (wa && sizeof(rig_id) != 0)
```

Comment:
`sizeof(rig_id)` mistake again. You're asking for the size of a char* (i.e. will always be 8).